tests/api/test.js

```diff
@@ -0,0 +1,303 @@
+let {GET, POST, DELETE, all, contains, json_equal, rand, noError, explode } = require("./utils")
```

Comment:
I don't like that these tests are depending on previous ones. I think we should do smth like `coffer delete / -r` before running each test (like in `bats` tests). + you can look [here](https://github.com/serokell/coffer/pull/32#discussion_r820250737) and write missing tests.

Reply:
Will do. I wondered how can I clear the state entirely, thanks!

Reply:
> + you can look here and write missing tests

I don't think we need to test every edge case for the Web API as we did for the CLI. Since they use the same core modules, that seems like duplicated effort.

We should treat it as "just another interface" for coffer's core, and test it as such: 
* make sure all inputs (URI path, query params, headers, etc) are correctly parsed and processed
* make sure all outputs (including error scenarios) are correctly formatted

Testing the output of error scenarios does involve testing a lot of edge cases, but perhaps not as many as we did in the CLI, I hope?